src/utils/fetchSolversInfo.ts

```diff
@@ -0,0 +1,66 @@
+export type SolverInfo = {
+  address: string
+  name: string
+  environment: string
+}
+
+export type SolversInfo = SolverInfo[]
+
+const SOLVER_SOURCE_PER_NETWORK = {
+  1: 'https://raw.githubusercontent.com/duneanalytics/spellbook/main/models/cow_protocol/ethereum/cow_protocol_ethereum_solvers.sql',
+  100: 'https://raw.githubusercontent.com/duneanalytics/spellbook/main/deprecated-dune-v1-abstractions/xdai/gnosis_protocol_v2/view_solvers.sql',
```

Comment:
Might be worth noting that the two different dune platforms were using here are both slightly different variations of a similar pattern I suppose the regex matching would capture both so it doesn't matter which kind of file is being read.